examples/config.yaml

```
@@ -10,3 +10,5 @@ backups:
   - kind: github/release
     from: orgs/SierraSoftworks
     filter: repo.public && !release.prerelease && !artifact.source-code
+  - kind: github/star
+    from: users/notheotherben
```

It's a small vanity thing, but why not this 😉 

```suggestion
    from: users/cedi
```

---

I had it at `users/cedi` initially, but changed it to your username for consistency with all the other unit-tests